refinery/config/settings/base.py

```diff
@@ -658,7 +658,8 @@ def get_setting(name, settings=local_settings, default=None):
 # Time in seconds to wait before killing unused visualization
 DJANGO_DOCKER_ENGINE_SECONDS_INACTIVE = 60 * 60
 # Location of DjangoDockerEngine proxy logging
-PROXY_LOG = '/tmp/django_docker_engine.log'
+DJANGO_DOCKER_ENGINE_DATA_DIR = '/data/django-docker-engine-data'
```

Comment:
`/data` doesn't exist in Vagrant VM and will go away on AWS once Refinery switches to storing data files in S3

Reply:
This is where docker instances will have mounted volumes for their inputs and ephemeral storage needs. The path is accessed by docker-engine: If docker-engine is running on a separate instance, it will be a directory on that instance. If we can use ECS, then this may be moot. But it needs to be somewhere for now, and this is better than having it hard-coded to a path on the root volume.

Using `/data` also seemed better than mounting an entirely new EBS volume.

On vagrant, do you have a strong preference about whether we create a `/data` directory as part of setup, or have this setting depend on environment, or something else entirely?

Reply:
OK then it's fine to use '/data' for Docker files for now on a condition that only django_docker_engine would have access to it to avoid tight coupling with the rest of the application. Also, django_docker_engine should consider this storage ephemeral because this is going to be the case when it runs on a separate instance.

It's probably a good idea to add `/data` to Vagrant for consistency and move all dirs from $project_root (`media`, `import`, `isa-tab` and Solr index dirs) there.

Finally, it'd be great to consolidate all storage related Puppet resources into one module (similar to [python.pp](https://github.com/refinery-platform/refinery-platform/blob/mccalluc/django-docker-data-dir/deployment/refinery-modules/refinery/manifests/python.pp) for example).